### DISCUSSION TITLE
Confirmed working on Arch

### DISCUSSION THREAD

**Discussion by Gerto** [tobias-kuendig/hacompanion#14] [General]
Hi,

Thanks for this great hacompanion!
Just wanted to confirm this is working perfectly on Arch Linux with KDE Plasma.

The only issue I ran into was in the systemd unit file "multi-user.target" does not exist for a user service which caused error message “non-existent unit multi-user.target”, so I had to replace that with "default.target".

I also added a custom "idle time" sensor with xprintidle which is also working perfectly.

Thanks!

**Comment by ecto1a2003** [comment]
was that for the steamdeck by chance?

**Reply by Gerto** [reply]
My original post was not for steam deck, however I have just installed it on Steam Deck and can also confirm it's working perfectly (once again, with "default.target")
